corporate/lib/stripe.py

```diff
@@ -1698,7 +1698,7 @@ def __init__(
     @override
     @property
     def billing_session_url(self) -> str:
-        return "TBD"
+        return f"{settings.EXTERNAL_URI_SCHEME}{settings.SELF_HOSTING_MANAGEMENT_SUBDOMAIN}.{settings.EXTERNAL_HOST}/realm/{self.remote_realm.uuid}"
 
```

Comment:
Do we need to add new URL for this to work for upgrade page via gear menu?

Reply:
No; the gear menu option for self-hosted systems does a fancy series of steps to reach here, but the initial URL is just always something like `/self-hosting`.